addon/PageScraper.js

```diff
@@ -73,6 +73,7 @@ PageScraper.prototype = {
           };
           resolve([imageWithSize]);
         });
+        image.addEventListener("error", () => reject());
```

Comment:
I think this is what we want, since ```the error``` event will fire only fire if something goes wrong while loading, which we probably want to capture. But we could also resolve with an empty array. What do you think? @Mardak